### DISCUSSION TITLE
Modify the font size of the logo on the app bar

### DISCUSSION THREAD

**Discussion by hsw1805** [AgainIoT/Open-Set-Go#53] [Feature Suggestion / Share Ideas]
![image](https://github.com/AgainIoT/Open-Set-Go/assets/66340991/3e252ed2-4bad-4f3d-a0c3-fa2032be9961)
디자인 부분 측면에서 가시성이 떨어지는 거 같습니다. 바꾸면 좋을 거 같아요.

**Comment by ymw0407** [comment]
의견주셔서 감사합니다! 해당 의견은 AgainIoT/Open-Set-Go_client#135 에서 반영 되어 적용되었습니다! Open-Set-Go v2.0에서 확인하실 수 있을 겁니다! 첫번째 Discussion 감사드립니다!!
![image](https://github.com/AgainIoT/Open-Set-Go/assets/77202633/b3a66168-5917-43dc-a476-eaf883de44d4)
